docs/network-flow-visibility.md

```diff
@@ -754,6 +755,15 @@ Mouse out or click on the background will bring all the traffic back.
 
 <img src="https://downloads.antrea.io/static/08032022/flow-visibility-np-3.png" width="900" alt="Network-Policy Flows Dashboard">
 
+#### Network Topology Dashboard
+
+Network Topology Dashboard visualizes Pod-to-Pod and Pod-to-Service traffic.
+Pods labels are visually grouped by Node. Each arrow points from the source to
+the destination, and is labelled by the cumulative amount of data transmitted
+in the selected time range.
+
+<img src="https://user-images.githubusercontent.com/10016630/220810363-96de054a-fee4-4e94-8c06-26d204f4c166.png" width="900" alt="Network Topology Dashboard service dependency graph">
```

Comment:
If you agree to hide the system-level default flows, better to replace this screenshot with some self-generated workload flows.

you can use this one:
```
kubectl label nodes k8s-node-control-plane app=control-plane
kubectl apply -f https://raw.githubusercontent.com/heanlan/antrea/e4c13b4c6788f1f1d75b4f412399eebba5a6c750/build/yamls/traffic3.yml
```